django_datawatch/tests/test_querysets.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.test.testcases import TestCase
+
+from django_datawatch.models import Result
+
+User = get_user_model()
+
+class ResultQuerySetTestCase(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(**{User.USERNAME_FIELD: 'test_user'})
+        self.group = Group.objects.create(name='test_group')
+
+    def test_for_user(self):
+        result_without_users_and_groups = Result.objects.create(slug='test_slug', identifier='test_identifier')
```

Comment:
We could have all of the following combinations.
Does testing all of these make sense to you?

* No group and user
   * Test positive when user has any group assigned
   * Test positive when user has no groups assigned at all
* One group but no user
   * Test positive when user is part of that group
   * Test negative when user is not part of that group
* Multiple groups but no user
   * Test positive when user is part of one of these groups
   * Test negative when user is not part of any of these group
* One user but no group
   *  Test positive when user is the assigned one
   *  Test positive when user is not the assigned one
* Multiple users but no groups
   * Test positive when user is one of the assigned users
   * Test negative when user isn't one of the assigned users
* One group and one user
   * Test positive when user is part of that group
   * Test positive when user is the assigned one
   * Test negative when user is neither part of that group nor the assigned user
* Multiple groups and one user
   * Test positive when user is part of one of these groups
   * Test positive when user is the assigned one
   * Test negative when user is neither part of these groups nor the assigned user
* One group and multiple users
   * Test positive when user is part of that group
   * Test positive when user is one of the assigned users
   * Test negative when user is neither part of that group nor one of the assigned users